lib/templates/deploy.bash.ruby.template

```diff
@@ -9,11 +15,25 @@ initializeDeploymentConfig() {
 		OPTIONS="--without $BUNDLE_WITHOUT";
 		echo "Bundle install with options $OPTIONS";
 	fi
+	
+	if [ -z $BUNDLE_INSTALL_LOCATION ]; then 
```

Comment:
Are all of these env vars documented? I feel like they should at least include "RUBY" somewhere in the name to help disambiguate.